### DISCUSSION TITLE
Support QOA audio format

### DISCUSSION THREAD

**Discussion by CodeLazier** [godotengine/godot-proposals#6533] [Engine Core]
Consider supporting support for audio file format QOA for faster decoding (3x faster than OGG) and better quality.

https://qoaformat.org/

**Comment by AThousandShips** [comment]
Considering this format has just been announced and the specification isn't even released yet I think it'll take a while until this format has enough support and use to justify including it, adding a format is only really reasonable when it has some kind of large scale use, not preemptively

**Reply by CodeLazier** [reply]
I agree with your, and believe that the performance, quality, and versatility are worth further observation.

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/3726.

Like QOI, it needs strong enough adoption before this can be considered for core. https://github.com/godotengine/godot-proposals/issues/1190 is more likely to be implemented in comparison.
